src/main/java/net/snowflake/ingest/streaming/internal/ParquetBufferValue.java

```diff
@@ -9,6 +9,8 @@ class ParquetBufferValue {
   // Parquet uses BitPacking to encode boolean, hence 1 bit per value
   public static final float BIT_ENCODING_BYTE_LEN = 1.0f / 8;
 
+  public static final float REPETITION_LEVEL_ENCODING_BYTE_LEN = 2.0f / 8;
```

Comment:
add comment on why this is 2.0f/8

Reply:
Added to where the definition level encoding is defined.